### PR TITLE
handle null values in scopes

### PR DESCRIPTION
**Pull request by fiji-flo**

### CODE REVIEW

File: src/assets/js/scope.js [hunk 1]
```diff
@@ -13,11 +13,11 @@ class Scope {
     const {
       staffInformation: { staff: { value: isStaff = false } = {} } = {},
       accessInformation: {
-        mozilliansorg: { values: { nda: isNdaed = false } = {} } = {},
+        mozilliansorg: { values: moziiliansorgGroups = {} } = {},
```
Comment (fiji-flo) [inline src/assets/js/scope.js]: `values` can be `null` and that is valid profile v2. That's what was happening here.

File: src/assets/js/scope.js [hunk 2]
```diff
@@ -13,11 +13,11 @@ class Scope {
...
       } = {},
-    } = user;
+    } = user || {};
```
Comment (fiji-flo) [inline src/assets/js/scope.js]: This assignment must not error on any valid profile v2 data.

File: src/assets/js/scope.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (fiji-flo) [reply src/assets/js/scope.js]: Yes, I’ll fix it when I’m back at the hotel. Otherwise fine?

Comment (gdestuynder) [reply src/assets/js/scope.js]: did you mean to type `moziiliansorgGroups` or `mozilliansorgGroups`?

Comment (gdestuynder) [inline src/assets/js/scope.js]:
```suggestion
        mozilliansorg: { values: mozilliansorgGroups = {} } = {},
```

File: src/assets/js/scope.js [hunk 3]
```diff
@@ -13,11 +13,11 @@ class Scope {
...
     this.isStaff = isStaff;
-    this.isNdaed = isNdaed;
+    this.isNdaed = 'nda' in (moziiliansorgGroups || {});
```
Comment (gdestuynder) [inline src/assets/js/scope.js]:
```suggestion
    this.isNdaed = 'nda' in (mozilliansorgGroups || {});
```

File: src/assets/js/scope.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (andrew-sunada) [reply src/assets/js/scope.js]: Yup! Fix that, and I'm good!

**Pull request merged**